### ISSUE TITLE
Option to generate constant name in uppercase

### ISSUE THREAD

**Issue by vithort** [kreuzerk/svg-to-ts#241]
_Constants can be declared with uppercase or lowercase, but a common convention is to use **all-uppercase** letters, especially for primitives because they are truly immutable._

Based on this, I would like to know if it is possible to generate the name of constants in **svg-to-ts-constants** in uppercase.

I can define the prefix of the constant with _option: prefix_, but I couldn't change the "name", is there any way? Could it be an improvment?

```
export const prefix_Dinossaur: {
  name: 'dinossaur';
  data: string;
} = {
  name: 'dinossaur',
  data: `<svg width="80" ...</svg>`
};
```

thank you! :)

**Comment by kreuzerk** [comment]
Yes, this is a valid feature request. Thx for bringing this up 👍
